.github/workflows/run_android_libftdi.yml

```diff
@@ -0,0 +1,74 @@
+name: Run libFTDI Android NDK
```

Comment:
if I am not mistaken wo libftdi boards which use serial port do not work, I think we can keep it in the same workflow